diff --git a/test/duplicate_test.go b/test/duplicate_test.go
index 083dbf5c..8779ce53 100644
--- a/test/duplicate_test.go
+++ b/test/duplicate_test.go
@@ -38,7 +38,7 @@ func TestDuplicatePodTaskRun(t *testing.T) {
 	defer tearDown(t, c, namespace)
 
 	var wg sync.WaitGroup
-	for i := 0; i < 25; i++ {
+	for i := 0; i < 10; i++ {
 		wg.Add(1)
 		taskrunName := fmt.Sprintf("duplicate-pod-taskrun-%d", i)
 		t.Logf("Creating taskrun %q.", taskrunName)
